examples/healthcare/plugins/entities/Patients/conformance/final/writer/writer.xqy

```diff
@@ -0,0 +1,22 @@
+xquery version "1.0-ml";
+
+module namespace plugin = "http://marklogic.com/data-hub/plugins";
+
+declare option xdmp:mapping "false";
+
+(:~
+ : Writer Plugin
+ :
+ : @param $id       - the identifier returned by the collector
+ : @param $envelope - the final envelope
```

Comment:
Will the upstream processes always be envelopes? I get that they might be now, but is that the only kind of transform you intend to support? If you have broader capabilities in mind, then maybe

  param $input - the conformed input